flink-staging/flink-streaming/flink-streaming-core/src/test/java/org/apache/flink/streaming/api/functions/PrintSinkFunctionTest.java

```diff
@@ -73,21 +51,22 @@ public void testPrintSinkStdOut(){
 		try {
 			printSink.open(new Configuration());
 		} catch (Exception e) {
-			e.printStackTrace();
+			Assert.fail();
 		}
 		printSink.setTargetToStandardOut();
 		printSink.invoke("hello world!");
 
 		assertEquals("Print to System.out", printSink.toString());
-		assertEquals("hello world!", stream.result);
+		assertEquals("hello world!\r\n", baos.toString());
```

Comment:
This will work on Windows (`\r\n` for newline) but not on Unix systems (`\n` for newline).